storage/innobase/log/log0log.cc

```diff
@@ -62,7 +62,7 @@ Created 12/9/1995 Heikki Tuuri
 
 #include "system_key.h"
 
-static redo_log_encrypt_enum found_log_encryption_mode = REDO_LOG_ENCRYPT_OFF;
+redo_log_encrypt_enum existing_redo_encryption_mode = REDO_LOG_ENCRYPT_OFF;
```

Comment:
lets follow the running style used. So have the name log_existing_encryption_type /mode.

Also add a declaration in header log0log.h